docs/getting-started.md

```diff
@@ -20,7 +20,7 @@ you'll need to acquire them on your own.
 
 !!! attention
 
-    The XQEMU project does not endorse or promote piracy. We don't link to the
+    The XQEMU project does not endorse or promote piracy. We don't link to
     copyrighted files, or discuss how to acquire them. The best way to acquire
```

Comment:
While we are at it: "best way" &rarr; "only legal way"?

Reply:
Agree